support-frontend/app/controllers/Application.scala

```diff
@@ -8,7 +8,7 @@ import assets.{AssetsResolver, RefPath}
 import com.gu.i18n.CountryGroup
 import com.gu.i18n.CountryGroup._
 import com.gu.identity.model.{User => IdUser}
-import com.gu.support.catalog.{SupporterPlus, TierThree}
+import com.gu.support.catalog.{Collection, HomeDelivery, Paper, SupporterPlus, TierThree}
```

Comment:
We'll need to do the same for Digital Subscriptions for promos to work on the Generic Checkout for that product too right?

Reply:
We can look at that in a follow-up PR